python3/totoro_agent/brain/pickup_tracker.py

```diff
@@ -10,6 +10,60 @@ def __init__(self):
     def update(self, game_state):
         # TODO gaurav
         # powerup - stop picking up powerups as soon as  diameter reaches double map size
```

Comment:
This one's just a note for whoever's doing the pick up algorithm. We want to prioritise the ammo as soon as the player diameter is great double the map size so that our bot can snipe the other bot from across the map

Reply:
There is no prioritise thing yet though, how can I do that.